### PR TITLE
Varied cleanup

### PR DESCRIPTION
**Pull request by gonuke**
Addresses issues #10, #15, #17, #19 

I'll rebase when necessary

### CODE REVIEW

File: src/report.tex [hunk 1]
```diff
@@ -688,20 +688,22 @@ \subsubsection*{Auxilliary Codes}
...
+  Code & Validation/Verification Status & References \\    
   \hline
-  pyne.mesh & Validated by unit tests & - \\
-  pyne.r2s & Validated & \cite{pyne_r2s} \\
-  pyne.alara & Validated & \cite{pyne_r2s} \\
-  pyne.mcnp & Validated & \cite{pyne_r2s} \\
-  pyne.source\_sampling & Validated & \cite{pyne_r2s} \\
+  pyne.mesh & Verified by unit tests & n/a \\
+  pyne.r2s & Validated & \cite{biondoFED2016} \\
+  pyne.alara & Verified by unit tests & \cite{biondoFED2016}  \\
```
Comment (makeclean) [inline src/report.tex:711]: @article{Biondo201677,
title = "Shutdown dose rate analysis with {CAD} geometry, Cartesian/tetrahedral mesh, and advanced variance reduction ",
journal = "Fusion Engineering and Design ",
volume = "106",
number = "",
pages = "77 - 84",
year = "2016",
note = "",
issn = "0920-3796",
doi = "http://dx.doi.org/10.1016/j.fusengdes.2016.03.004",
url = "http://www.sciencedirect.com/science/article/pii/S0920379616302009",
author = "Elliott D. Biondo and Andrew Davis and Paul P.H. Wilson",
keywords = "Shutdown dose rate",
keywords = "CAD",
keywords = "Tetrahedral mesh",
keywords = "Variance reduction "
}

bibtex entry for biondos article

**Comment by makeclean** [comment]
I think only add the bibtex entry and thats it.

**Comment by gonuke** [comment]
Hold on this!!!

**Comment by gonuke** [comment]
OK - added a printed note to call out what we need from Ahmad - ready to go.

**Comment by makeclean** [comment]
Superb.

**Pull request merged**